src/gui/optionsdialog.ui

```diff
@@ -1631,13 +1618,26 @@
                  </item>
                 </layout>
                </item>
+               <item>
+                <widget class="QCheckBox" name="checkProxyPeerConnecs">
+                 <property name="enabled">
+                  <bool>false</bool>
+                 </property>
+                 <property name="toolTip">
+                  <string>Web seeds as well as regular bittorrent peer connections are considered as peer connections, tracker connections and DHT traffic are not.</string>
```

Comment:
This sounds to me like tracker connections and DHT traffic won't be proxied.

Reply:
I think "as" is redundant. I think "as well as" could be simplified to "and".
there is a separate setting affecting whether trackers are proxied or not